recipes/hapcut2/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: hapcut2
+  version: '1.1'
```

Comment:
Maybe add package name and version as variable in the reipe:
```
{% set name = "hapcut" %}
{% set version = "1.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}
```

Reply:
I’ll change it to be more consistent with other recipe. (Personally, I think this just adds clutter. The only advantage is that I don’t have to change the URL when bumping the version manually, but then I’ll have the file open anyway because I need to copy and paste the new checksum.)